.github/workflows/ci.yml

```diff
@@ -3,15 +3,16 @@ name: testsuite
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, migrate-to-gcp ]
```

Comment:
Oh, and this is a reminder to go revert this change and all other changes for allowing you to test the actual deploy flow on this branch.